### ISSUE TITLE
question

### ISSUE THREAD

**Issue by kaishijeng** [ElectronicElephant/openpilot-reimplementation#2]
Is your repository related to this https://github.com/caizhitian/openpilot-reimplementation?

Thanks,

**Comment by ElectronicElephant** [comment]
Yes, they are related. Please wait for another one or two weeks before we submitting the report to arXiv and get everything done. Thank you.
